deps/build.jl

```diff
@@ -1,15 +1,37 @@
-#
-
-find_llvm() = begin
+#Returns the path as determined by llvm-config, or throws an error if LLVM
+#could not be found
+function find_llvm()
     if haskey(ENV, "TRAVIS")
         return readchomp(`which llvm-config-3.4`)
     end
     tmp = joinpath(JULIA_HOME, "llvm-config")
```

Comment:
It seems that the julia-0.5 build provided by @staticfloat's PPA is simply too old (20160408) and does not support the `@static` macro syntax. Will re-test when binaries get updated.